roqoqo/tests/integration/operations/multi_qubit_gate_operations.rs

```diff
@@ -60,7 +60,7 @@ fn test_circuit_multi_ms(qubits: Vec<usize>) {
 }
 
 #[test_case(vec![0,1]; "two_qubit")]
-fn test_matrix_output(qubits: Vec<usize>) {
+fn test_matrix_output_multi_ms(qubits: Vec<usize>) {
```

Comment:
Why all the renames ?

Reply:
To have a consistent notation. All other tests refer to the gate they are testing in their name.